### ISSUE TITLE
Can't join previous projects

### ISSUE THREAD

**Issue by TechSupport26** [lightningrodlabs/acorn#277]
v5.0.2

I am not able to join projects. (I don't think there is anything else to say.) Let me know if you need more details.

![image](https://user-images.githubusercontent.com/65631811/235548521-3196e2c3-a24b-436f-83e2-308cc246a62f.png)

### Computer Specs

Operating System: Debian GNU/Linux 12
KDE Plasma Version: 5.27.2
KDE Frameworks Version: 5.103.0
Qt Version: 5.15.8
Kernel Version: 6.1.0-6-amd64 (64-bit)
Graphics Platform: X11
Processors: 4 × Intel® Core™ i7-7500U CPU @ 2.70GHz
Memory: 7.5 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 620
Manufacturer: Dell Inc.
Product Name: XPS 13 9360

**Comment by Connoropolous** [comment]
ok noted. we will try to reproduce

**Comment by Connoropolous** [comment]
We've been able to reproduce, so we're looking into a fix

**Comment by Connoropolous** [comment]
@TechSupport26 
can you describe more about the project? 
Who does the project belong to? Are you trying to join the project of a peer? or is it also your own project somehow? (if so, why join your own project?)

**Comment by TechSupport26** [comment]
I tried to import a project file .json that I downloaded a few months ago from v-1.0.3 and it just sits there loading.

**Comment by Connoropolous** [comment]
from v-1.0.3 ?

**Comment by TechSupport26** [comment]
It was around that time that I made the backup.

**Comment by Connoropolous** [comment]
got it. That is likely to old a version for us to have tested on and supported as a migration.

**Comment by TechSupport26** [comment]
Fair enough

**Comment by TechSupport26** [comment]
So I have just gone back a few versions and imported some old project files and walked them through the versions and they came across fine till version 4.0.0 where I had to copy the secret phrase from version 3.0.0 and join the project. I can't get them to version 5.0.2 but that is a separate issue.

I found it cool that I was able to manually do that.
I am going to close this as the original issue was based off of joining a project from previous versions. As that was not intended to be used in the way that I used it.
